CreatubblesAPIClient/Sources/Model/Creation.swift

```diff
@@ -189,6 +191,9 @@ open class Creation: NSObject, Identifiable {
 
         contentType = creationEntity.contentType ?? ""
 
+        let entityTags: Array<String> = creationEntity.tags.flatMap { $0.tag }
+        tags = entityTags.isEmpty ? nil : entityTags
```

Comment:
Would it make sense to return an empty array here (in the case of `abilities` and `createdAtAgePerCreator` we do provide fallback values)?